hw/ip/i2c/data/i2c.hjson

```diff
@@ -291,6 +291,20 @@
                 When 0, the bus monitor will report that the bus is always free, so the controller FSM is never blocked from transmitting.
                 '''
         }
+        { bits: "6"
+          resval: "0"
+          name: "TX_STRETCH_CTRL_EN"
+          desc: '''
+                Enable setting !!TARGET_EVENTS.TX_PENDING to gate target read transfers.
```

Comment:
nit: imo this first line of the description is a bit too terse, I had to think quite hard to parse it the first time reading. L300 would be a better first line for the description I think.

Reply:
Works for me. I deleted this line and made the one you selected be first.